Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/cherry_pick.py

```diff
@@ -41,6 +42,12 @@ def parser(cls, parser, loggers=None):
             type=str, default=None,
             help='String representation of a commit to be cherry-picked',
         )
+        parser.add_argument(
+            '--resolve', '--no-resolve',
```

Comment:
Not sure if `resolve` is a good name for this, the help description doesn't seem to clarify it much.

Reply:
Maybe `--use-original` and `--no-original`?